app/src/main/java/woowacourse/movie/presentation/home/HomeActivity.kt

```diff
@@ -0,0 +1,64 @@
+package woowacourse.movie.presentation.home
+
+import android.os.Bundle
+import android.view.MenuItem
+import androidx.appcompat.app.AppCompatActivity
+import androidx.databinding.DataBindingUtil
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.commit
+import woowacourse.movie.R
+import woowacourse.movie.databinding.ActivityHomeBinding
+import woowacourse.movie.presentation.homefragments.movieList.MovieListFragment
+import woowacourse.movie.presentation.homefragments.reservation.ReservationFragment
+import woowacourse.movie.presentation.homefragments.setting.SettingFragment
+
+class HomeActivity : AppCompatActivity() {
+    private lateinit var binding: ActivityHomeBinding
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_home)
+        binding = DataBindingUtil.setContentView(this, R.layout.activity_home)
+
```

Comment:
> Use this version only if `layoutId` is unknown in advance. Otherwise, use the generated Binding's inflate method to ensure type-safe inflation.

찾아보니 `DataBindingUtil` 은 사전에 layoutId를 알지 못하는 상황에서 해당 layout을 참조해야 할 때 사용하는 것이네요! 
Binding class를 사용하도록 수정해보겠습니다 🫡